NuGet.Config

```diff
@@ -10,6 +10,8 @@
         <clear />
     </disabledPackageSources>
     <config>
+        <!-- TODO: TEMPORARY UNTIL PGO-Helpers is updated to search both. We shouldn't need to keep the globals path here. -->

```

Comment:
bare TODO

Reply:
Waiting on @dkbennett to fix the package on this one.

cc: @kmahone

Reply:
cc: @bartekk8 is what I actually meant... Sorry.